pygmt/base_plotting.py

```diff
@@ -440,6 +440,204 @@ def grdview(self, grid, **kwargs):
                 arg_str = " ".join([fname, build_arg_string(kwargs)])
                 lib.call_module("grdview", arg_str)
 
+    @fmt_docstring
+    @use_alias(
+        R="region",
+        J="projection",
+        B="frame",
+        C="cmap",
+        U="timestamp",
+    )
+    @kwargs_to_strings(R="sequence", i="sequence_comma")
+    def velo(self, x=None, y=None, data=None, scaling=None, **kwargs):
+        """
+        Plot velocity vectors, crosses, and wedges
+
+        Reads data values from files [or standard input] and will plot velocity
+        arrows on a map. Most options are the same as for plot, except -S.
+
+        Must provide either *data* or *x* and *y*, and *scaling*.
+
+
+        Full option list at :gmt-docs:`supplements/geodesy/velo.html`
+
+        {aliases}
+
+        Parameters
+        ----------
+        x/y : float or 1d arrays
+            The x and y coordinates, or arrays of x and y coordinates of the
+            data points
+        data : str or 2d array
+            Either a data file name or a 2d numpy array with the tabular data.
+            Use option *columns* (i) to choose which columns are x, y, color,
+            and size, respectively.
+
+        {J}
+
+        {R}
+
+        scaling: str
+            Selects the meaning of the columns in the data file and the figure
+            to be plotted. In all cases, the scales are in data units per length
+            unit and sizes are in length units (default length unit is
+            controlled by `PROJ_LENGTH_UNIT <https://docs.generic-mapping-tools.org/latest/gmt.conf.html#term-proj-length-unit>`_
+            unless **c**, **i** , or **p** is appended).
+
+            "**-Se**\ *velscale/confidence*\ [\ **+f**\ font]"
```

Comment:
I didnt find any other way than using '\ ' to be able to have different formating in the same word.
The idea is to reproduce the [original gmt formatting of the argument](https://docs.generic-mapping-tools.org/latest/supplements/geodesy/velo.html?highlight=velo#required-arguments) :
<strong>-Se</strong><em>velscale/confidence</em>[<strong>+f</strong><em>font</em>]

Reply:
See my comment on using the raw-string. Also, it would be great if you can leave out the **`-S`** (i.e. just use "**e**"), since the `velo` user will pass in the argument as `S="e..."` rather than `S="Se..."`.

Reply:
If I understand this correctly, the Python code style checker thinks that you want to use an escape sequence within the docstring you're writing. Since Python actually doesn't need to parse the `\ `, but only Apache/RST, it might be sufficient to either add another backslash (`\\ `), or make the entire docstring a raw one by prepending an "r" in front of it (`r""" ...`). I don't know what's better style in this particular case.

That being said, I think consistency is the more important than individual formatting tweaks, and given that [plot](https://www.pygmt.org/dev/api/generated/pygmt.Figure.plot.html#pygmt.Figure.plot) and all the other functions don't use the bolding when it comes to the argument call signature, I'd also not include it in the first place.

Reply:
Thanks for chipping in @tobiscode (sorry for beating you to it :stuck_out_tongue_closed_eyes: ). On the point that the other functions don't use **bold**, I agree that using backticks to format the code and keep things consistent is good. That said, the GMT **bold** style formatting here does look nice, and I'm open to applying either.

Reply:
Thank you @weiji14 @tobiscode for your replies. I will make the change using the raw string. I quite like the GMT **bold** and *italic* formating here as it make it easier to understand how is "coded" the parameter.